src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePrev = sourceString.trim().split(';');
+
+  const style = stylePrev
+    .filter((elm) => {
+      if (elm.indexOf(':') !== -1) {
+        return elm;
+      }
```

Comment:
The filter callback should return a boolean value. Instead of returning the element itself, return a boolean expression like 'elm.indexOf(':') !== -1'.